test/BUILD

```diff
@@ -0,0 +1,105 @@
+cc_library(
+    name = "output_test_helper",
+    srcs = [
+        "output_test.h",
+        "output_test_helper.cc",
+    ],
+    deps = ["//:benchmark"],
+)
+
+cc_binary(
```

Comment:
I wish they could, but we don't run googletest tests (which cc_test expects, as i understand it).